config.yaml

```diff
@@ -24,9 +24,9 @@ jobfs: 1500MB
 # Modules for loading model executables
 modules:
   use:
-      - /g/data/vk83/modules
+      - /g/data/vk83/prerelease/modules
   load:
-      - access-esm1p5/2024.05.1
+      - access-esm1p6/pr26-4
```

Comment:
We aren't going to pr27?

Reply:
Good question, that will be the next step. This PR only brings in the wombatlite and cable 3 changes, matching the executables in pr26. 

The executables in pr27 need the iceberg files and namelist changes, and so I'll make all those changes after this.